src/main/java/stsjorbsmod/actions/SowAction.java

```diff
@@ -0,0 +1,38 @@
+package stsjorbsmod.actions;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.DamageAllEnemiesAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardBrieflyEffect;
+import stsjorbsmod.cards.cull.ReapAndSow;
+import stsjorbsmod.patches.ActionShouldPersistPostCombatField;
+
+public class SowAction extends AbstractGameAction {
+    private int sowDamage;
+
+    public SowAction(int sowDamage) {
+        this.sowDamage = sowDamage;
+        ActionShouldPersistPostCombatField.shouldPersistPostCombat.set(this, true);
+    }
+
+    public void update() {
+        AbstractCard card = new ReapAndSow();
```

Comment:
It might have been better to pass along or get a copy of an instance of ReapAndSow from the deck, but this is fine.